src/redux/reducers/differentialExpression/index.js

```diff
@@ -39,6 +41,9 @@ const differentialExpressionReducer = (state = initialState, action) => {
     case EXPERIMENT_SETTINGS_PIPELINE_START: {
       return initialState;
     }
+    case CELL_SETS_DELETE: {
+      return differentialExpressionUnselectDeletedOption(state, action);
```

Comment:
The naming we are using for reducers mean this should just be `cellSetsDelete`

Reply:
Think of it as one single reducer being split up across slices

Reply:
Ahh... ok

Reply:
reducer renamed